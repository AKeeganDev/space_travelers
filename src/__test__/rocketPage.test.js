import React from 'react';
import renderer from 'react-test-renderer';

describe('renders page to DOM and we test functionality', () => {
  it('renders the Rocketcard Correctly in the DOM', () => {
    const tree = renderer
      .create(<rockets />)
      .toJSON();
    expect(tree).toMatchSnapshot();
  });
});
