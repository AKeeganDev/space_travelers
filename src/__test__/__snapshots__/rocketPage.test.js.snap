// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders page to DOM and we test functionality renders the Rocketcard Correctly in the DOM 1`] = `<rockets />`;
